packages/react-tinacms/src/use-form.ts

```diff
@@ -73,7 +74,7 @@ export function useCMSForm<FormShape = any>(
  * This hook is useful when dynamically creating fields, or updating
  * them via hot module replacement.
  */
-function updateFormFields(fields?: Field[], form?: Form) {
+function useUpdateFormFields(fields?: Field[], form?: Form) {
```

Comment:
Considering that’s just a convention, that’s a weird rule to enforce🙄 but that’s fine.

Reply:
Well, I'm just operating under the assumption that the eslint rules in place are what are desired. I do believe that react will do different stuff based on this naming, but I believe it's just to angrily report when you've violated the rules of hooks.

Reply:
I think it's just for the eslint rule. But yeah we might as well stick with convention. May want to consider renaming it in the future.

> **Do I have to name my custom Hooks starting with “use”?** 
> 
> Please do. This convention is very important. Without it, we wouldn’t be able to automatically check for violations of rules of Hooks because we couldn’t tell if a certain function contains calls to Hooks inside of it.

https://reactjs.org/docs/hooks-custom.html#using-a-custom-hook